private/buf/buffetch/internal/proto_file_ref.go

```diff
@@ -14,6 +14,8 @@
 
 package internal
 
+import "github.com/bufbuild/buf/private/pkg/app"
```

Comment:
Nit: import formatting with parens.

Reply:
talked in person, not worrying about this, this is acceptable